pkgs/applications/misc/octoprint/default.nix

```diff
@@ -107,6 +107,24 @@ let
           }
         )
 
+        # Built-in dependency
+        (
+          self: super: {
+            file-read-backwards = self.buildPythonPackage rec {
+              pname = "file-read-backwards";
+              version = "2.0.0";
+
+              src = pkgs.python38Packages.fetchPypi {
```

Comment:
Do we really need python38?

Reply:
Maybe @mweinelt can confirm, but I was having trouble building some of the dependencies of octoprint on python 3.9. The change was made in https://github.com/NixOS/nixpkgs/commit/fe6d9366c9e5425ee828b8436870f18868cd3d23. Reverting that change (if possible/necessary) is not relevant to this PR.

Reply:
ah nevermind, I have used `pkgs.python38Packages` instead of just using `super`. I'll change that.

Reply:
> Do we really need python38?

Yes. We currently need python38.